CHANGELOG

```diff
@@ -189,6 +190,8 @@ Version 3.6 (2018-03-23)
     Updates to the translations (DE, EN_GB, ES, FR, IT, NL, ZH_CN) [translation teams]
     all bridges don't relay postings anymore that are posted to a public forum [annando]
     DAV addon marked unsupported [tobiasd]
+    communityhome addon marked unsupported [MrPetovan]
+    yourls addon makrked unsupported [Benjamin Lorteau]
```

Comment:
Can you please use my pseudonym `MrPetovan` instead? I probably messed up the commit authorship at some point but I'd rather the Changelog to be consistent.

Reply:
Sure :-)